specs/Spec.Hash.Definitions.fst

```diff
@@ -154,35 +165,48 @@ let word_t: hash_alg -> Tot inttype = function
   | SHA2_384 | SHA2_512 -> U64
   | Blake2S -> U32
   | Blake2B -> U64
-  | SHA3_256 -> U64
+  | SHA3_224 | SHA3_256 | SHA3_384 | SHA3_512 | Shake128 | Shake256 -> U64
 
 inline_for_extraction
 let row (a:blake_alg) = lseq (uint_t (word_t a) SEC) 4
 
 inline_for_extraction
 let word (a: hash_alg) = match a with
-  | MD5 | SHA1 | SHA2_224 | SHA2_256 | SHA2_384 | SHA2_512 | SHA3_256 -> uint_t (word_t a) SEC
   | Blake2S | Blake2B -> row a
+  | _ -> uint_t (word_t a) SEC
```

Comment:
It might be useful to avoid wildcard uses in specs to reduce the risk of mistakes in the future.

Reply:
Fixed in 2ce8329cd6 which is part of the subsequent PR. Also fixed other wildcards below.